### ISSUE TITLE
A button to display/hide blame information on pull request review code line

### ISSUE THREAD

**Issue by lunny** [go-gitea/gitea#16306]
Most time, when review a pull request, I want to know if this PR should be back port to an old release version. So if we can know the changed line 's blame information. That's great and useful.

**Comment by jan-lhotka-almamedia** [comment]
Yes, we would also welcome this feature. After migrating from gitbucket, this is one of the few things we miss. Thanks
